src/contracts/CowProtocolToken.sol

```diff
@@ -4,12 +4,12 @@ pragma solidity ^0.8.10;
 import "./mixins/InflationaryToken.sol";
 import "./vendored/mixins/StorageAccessible.sol";
 
-/// @dev The governance token for the CowSwap protocol.
-/// @title CowSwap Governance Token
-/// @author Gnosis Developers
-contract CowSwapToken is InflationaryToken, StorageAccessible {
+/// @dev The governance token for the CoW Protocol.
+/// @title CoW Protocol Governance Token
+/// @author CoW Protocol Developers
+contract CowProtocolToken is InflationaryToken, StorageAccessible {
```

Comment:
Since it's a variable name and not a long-form text, I thought "CoW" was heavy and harder to read and type, and knowing that it's an acronym doesn't really give anything useful in the code. But overall I think it's a very minor concern, so both would be ok for me.

Reply:
> I thought "CoW" was heavy and harder to read and type

I also agree, just though I should ask the question 😛.